3-bull-jobs/finished-app/controllers/index.js

```diff
@@ -1,5 +1,3 @@
-const MovieController = require('./moviesController');
```

Comment:
I know we said ES6 but... any reason to change every single import?
Doing this change during the workshop will not be a gain I think...

Reply:
I just changed the imports for the bull finished app. Basically it happened because I started coding with imports from scratch (I forgot we where using requires). When I realized we where, it was less of an effort to change the requires to imports. I can make all requires if necessary but all is working well.

Reply:
Nothing wrong with the imports if it's working. But I'll still inclined to keep those in CommonJS. We are not really doing any dependency strategy neither and import doesn't not cache on module exports, so that is why I used.
Also the idea is start this over during the workshop from initial app and at the end have same result. Doing this changes won't be much of a content really

Reply:
Sure, I'll redo all import into requires then.